ui/src/app/common/services/user.service.ts

```diff
@@ -33,11 +34,11 @@ export class UserService {
 		});
 	}
 
-	public login(login: string, password: string) {
+	public login(login: string, password: string, loginSuccess: () => void = _.noop) {
```

Comment:
`loginSuccess: () => void = _.noop` looks like callback. It's better to use Promise or Observable instead

Reply:
done

Reply:
https://github.com/taraswww777/AngularGlobalMentoringProgram2019/commit/a3156cda